parastell/invessel_build.py

```diff
@@ -565,11 +575,19 @@ def __init__(
         self.poloidal_angles = poloidal_angles
         self.wall_s = wall_s
         self.radial_build = radial_build
+        self.separate_chamber = separate_chamber
 
-        self.plasma_mat_tag = 'Vacuum'
-        self.sol_mat_tag = 'Vacuum'
+        if self._separate_chamber:
+            self.plasma_mat_tag = 'Vacuum'
+            if self._wall_s > 1.0:
+                self.sol_mat_tag = 'Vacuum'
+        else:
+            self.chamber_mat_tag = 'Vacuum'
 
-        for name in kwargs.keys() & ('plasma_mat_tag', 'sol_mat_tag'):
+        for name in kwargs.keys() & (
+            'separate_chamber', 'plasma_mat_tag', 'sol_mat_tag',
```

Comment:
didn't we just explicitly set `separate_chamber` as a required argument?

Reply:
Yes, I forgot to remove this. I did not have `separate_chamber` as an explicit keyword argument before.